### DISCUSSION TITLE
Flags go to svg

### DISCUSSION THREAD

**Discussion by KG7x** [tltneon/lgsl#38] [Ideas]
How do you look at replacing flags with svg?
like this: https://github.com/lipis/flag-icon-css
4x3 or 1x1
svg can be gziped some flags less weight when png and scalable

**Comment by tltneon** [comment]
@KG7x 3 months passed :laughing: 
I also looking for replacements for default flags. Unfortunately, SVG much heavier and I don't want to place it into master branch of LGSL.
So I made custom script that can replace flags with SVG without downloading all the flag-icon-css package: https://github.com/tltneon/lgsl/blob/master/lgsl_files/scripts/flag-icon.js

**Reply by KG7x** [reply]
Жаль пока негде проверить. где тебя еще можно найти? ток форум ?

**Reply by tltneon** [reply]
Почта: neon1226@yandex.com
